src/content-handlers/iiif/modules/uv-contentleftpanel-module/TreeView.ts

```diff
@@ -63,37 +83,33 @@ export class TreeView extends BaseView<ContentLeftPanel> {
   }
 
   public selectNode(node: TreeNode): void {
-    if (!this.treeComponent.selectedNode) {
-      this.treeComponent.expandParents(node, true);
-
-      const link: Element | undefined = this.$tree.find(
-        "#tree-link-" + node.id
-      )[0];
-
-      if (link) {
-        // link.scrollIntoView({ inline: 'center' });
-      }
+    this.treeComponent.expandParents(node, true); // Expand node parents
+    const link: Element | undefined = this.$tree.find("#tree-link-" + node.id)[0];
+    if (link) {
+     // link.scrollIntoViewIfNeeded();
```

Comment:
I un-commented the initial code first, but didn't like how it looked as it was making the page jump down so changed it to this which worked better, but then I discovered a bug with this in certain manifests so commented it out as well - would it be better to just put it back to how it was?

Reply:
If you think this version is better, I don't think it hurts to change the comment... but maybe it would be worth adding another comment above this line explaining why it is commented out (e.g. "Bug X occurs when you load manifest Y, so this is commented out until we can find a fix.")